examples/phi2/README.md

```diff
@@ -1,13 +1,35 @@
 # Phi-2
 
+## Install the onnxruntime-genai library
+
+* (Temporary) Build the library according to [the build instructions](../README.md#build-from-source)
+
+* Install the python package
+
+  ```bash
+  cd build/wheel
+  pip install onnxruntime-genai-*.whl
+  ```
+
 ## Get the model
 
+Install the model builder script dependencies
+
 ```bash
-python -m onnxruntime_genai.scripts.export.py -m microsoft/phi-2 -e cpu -p int4 -o model/model.onnx
+pip install numpy
+pip install transformers
+pip install torch
+pip install onnx
+pip install onnxruntime
```

Comment:
is this still mandatory if `onnxruntime_genai` includes the `builder` script? having to manually install dependencies doesn't seem ideal.

Reply:
say I run `pip install onnxruntime_genai`. then I'd expect to be able to use it (including the builder) without installing additional dependencies.

I'm not so experienced with python packaging, but FWIW I think onnxruntime uses `setup(install_requires=...)` to do this.
https://github.com/microsoft/onnxruntime/blob/efbe2b84556c195e7d7f3353321eb3f410a1e645/setup.py#L695

Reply:
We're going to leave it as is for now, as those dependencies are not required for the genai runtime. We will try and think of a better way to do this though.